README.md

```diff
@@ -4,11 +4,13 @@
 
 NATS Monitoring, Simplified.
 
-NOTE:  This should be considered *alpha* and is currently a work in progress.
+NATS surveyor polls the NATS server for `Statz` messages to generate data for
+Prometheus.  This allows a single exporter to connect to any NATS server and
+get an entire picture of a NATS deployment without requiring extra monitoring
+components or sidecars.  Surveyor has been used extensively by [Synadia](https://synadia.com).
```

Comment:
also mention that it can do the traces not just statz?

Reply:
I updated the paragraph to add tracing.